data/bookkeeping/block.go

```diff
@@ -214,27 +214,27 @@ type (
 		NextProtocolSwitchOn basics.Round `codec:"nextswitch"`
 	}
 
-	// CompactCertState tracks the state of compact certificates.
-	CompactCertState struct {
+	// StateProofTrackingData tracks the status of state proofs.
+	StateProofTrackingData struct {
```

Comment:
Basically, I tend to agree that it is too general.
However, It is already implemented, and we would need to spend time to clean that up. More to that,
since one of our proirities in the near future is to have a rapid state proof chain we might need that.